Vonk.Plugin.DocumentOperation.Tests/DocumentOperationTests.cs

```diff
@@ -132,8 +130,8 @@ public async Task DocumentOperationPOSTReturn400OnMissingId()
 
             // Check response status
             testContext.Response.HttpResult.Should().Be(StatusCodes.Status400BadRequest, "$document should fail with HTTP 400 - Bad request if Parameters resource does not contain an id");
-            testContext.Response.Outcome.Should().NotBeNull();
-            testContext.Response.Outcome.Issue.Should().Contain(issue => issue.Code.Equals(VonkIssues.INVALID_REQUEST.Type));
+            testContext.Response.Outcome.Should().NotBeNull("At least one OperationOutcome should be returned");
+            testContext.Response.Outcome.Issues.Count(issue => issue.IssueType.Equals(VonkOutcome.IssueType.Invalid)).Should().NotBe(0, "Request should be rejected as an invalid");
```

Comment:
I found Issues.Should().Contain(...) easier to read that .Count().Should().NotBe(0).